npm/vite-dev-server/src/plugins/cypress.ts

```diff
@@ -76,7 +76,7 @@ export const Cypress = (
       indexHtmlContent = indexHtmlContent.replace(
         '<head>',
         `<head>
-        ${scriptTagsToInject.map((script) => script.toString())}
+        ${scriptTagsToInject.map((script) => script.toString()).join('')}
```

Comment:
Is the `toString` still necessary here since [join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#description) should call `toString` on the contents?

```suggestion
        ${scriptTagsToInject.join('')}
```

Reply:
That's clean I'll throw this in